README.md

```diff
@@ -112,7 +112,10 @@ may not be instantiated are greyed out):
 
 ![Image](doc/images/type_hierarchy.png?raw=true)
 
-### [Chisel Tutorial](https://github.com/ucb-bar/chisel-tutorial)
+### Learning Chisel
```

Comment:
What do you think of putting this right under "Getting Started"? I'd imagine that if one already knew how to use Chisel, one wouldn't bother with reading this README.

Reply:
I moved it a bit higher